gdpr-samples/gdpr-qsg/src/main/resources/oidc-config-notification-center.xml

```diff
@@ -24,7 +24,7 @@
                 <!--Optional:-->
                 <xsd1:applicationName>notification-center</xsd1:applicationName>
                 <!--Optional:-->
-                <xsd1:callbackUrl>http://localhost.com:8080/notification/oauth2client</xsd1:callbackUrl>
+                <xsd1:callbackUrl>http://localhost:8080/notification/oauth2client</xsd1:callbackUrl>
```

Comment:
can't we configure the hotstname as a variable here? e.g. http://${host.name}:8080/notification/oauth2client and then read the actual hostname from a single file as it is hardcoded in several places?

Reply:
This will be fixed in next milestone https://github.com/wso2/samples-is/issues/82